tool/flutter_ci_script_beta.sh

```diff
@@ -29,12 +29,13 @@ declare -ar PROJECT_NAMES=(
     "desktop_photo_search/fluent_ui"
     "desktop_photo_search/material"
     "dynamic_theme"
-    "experimental/federated_plugin/federated_plugin"
-    "experimental/federated_plugin/federated_plugin/example"
-    "experimental/federated_plugin/federated_plugin_macos"
-    "experimental/federated_plugin/federated_plugin_platform_interface"
-    "experimental/federated_plugin/federated_plugin_web"
-    "experimental/federated_plugin/federated_plugin_windows"
+    # TODO(ewindmill): package:html needs to be replaced by package:web
```

Comment:
I plan on removing or moving all the samples in `experimental` before landing this. Ignore this change for now, just testing CI